themes/bootstrap3/templates/myresearch/checkedout.phtml

```diff
@@ -1,91 +1,97 @@
         <? endif; ?>
-      <? endforeach; ?>
-
-      <? $i = 0; foreach ($this->transactions as $resource): ?>
-        <hr/>
-        <? $ilsDetails = $resource->getExtraDetail('ils_details'); ?>
-        <div id="record<?=$this->escapeHtmlAttr($resource->getUniqueId())?>" class="row result">
-          <? if ($this->renewForm): ?>
-            <? if (isset($ilsDetails['renewable']) && $ilsDetails['renewable'] && isset($ilsDetails['renew_details'])): ?>
-              <? $safeId = preg_replace('/[^a-zA-Z0-9]/', '', $ilsDetails['renew_details']); ?>
-              <div class="col-xs-1">
+      <? endif; ?>
+      <? if (isset($ilsDetails['renewable']) && $ilsDetails['renewable'] && isset($ilsDetails['renew_details'])): ?>
+        <? $safeId = preg_replace('/[^a-zA-Z0-9]/', '', $ilsDetails['renew_details']); ?>
+        <input class="pull-left flip" type="hidden" name="renewAllIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" />
+      <? endif; ?>
+    <? endforeach; ?>
+
+    <? $i = 0; foreach ($this->transactions as $resource): ?>
+      <? $ilsDetails = $resource->getExtraDetail('ils_details'); ?>
+      <div id="record<?=$this->escapeHtmlAttr($resource->getUniqueId())?>" class="result">
+        <? if ($this->renewForm): ?>
+          <? if (isset($ilsDetails['renewable']) && $ilsDetails['renewable'] && isset($ilsDetails['renew_details'])): ?>
+            <? $safeId = preg_replace('/[^a-zA-Z0-9]/', '', $ilsDetails['renew_details']); ?>
+            <div class="checkbox">
+              <label>
                 <input class="checkbox-select-item" type="checkbox" name="renewSelectedIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" id="checkbox_<?=$safeId?>" />
-                <input type="hidden" name="selectAllIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" />
-                <input type="hidden" name="renewAllIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" />
-              </div>
-            <? endif; ?>
+              </label>
+              <input type="hidden" name="selectAllIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" />
+              <input type="hidden" name="renewAllIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['renew_details'])?>" />
+            </div>
           <? endif; ?>
+        <? endif; ?>
 
-          <? $cover = $this->record($resource)->getCover('checkedout', 'small'); ?>
-          <? if ($cover): ?>
-            <div class="col-sm-2 col-xs-11 text-center">
-            <? /* Display thumbnail if appropriate: */ ?>
-              <?=$cover?>
+        <?
+          $coverDetails = $this->record($resource)->getCoverDetails('checkedout', 'medium', $this->recordLink()->getUrl($resource));
+          $cover = $coverDetails['html'];
+          $thumbnail = false;
+          $thumbnailAlignment = $this->record($resource)->getThumbnailAlignment('list');
```

Comment:
Does it make sense to use the list thumbnail alignment setting for holds/ILL/storage retrieval/checked out lists? Maybe, or maybe it would be better to create an 'account' category for these, with a new accountThumbnailsOnLeft setting in config.ini. Either way, we should document which setting controls this behavior -- either expanding the comment on the existing list setting to note that it will affect not just list thumbnails but also all account-related ones, or else introducing the new account-specific setting.